src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -98,11 +104,16 @@ public void run() {
             config
                 .subList(1, config.size())
                 .stream()
-                .map(a -> accessor(a, context, valueSet, localVars, null))
+                .map(a -> accessor(a, context, valueSet, localVars))
                 .collect(ImmutableList.toImmutableList()))
         .build();
   }
 
+  private static OutputView.VariableView accessor(
```

Comment:
Let's put all the utility functions (`accessor*`, `identifierEnd`) at the end of the fileso the various `*View()` are together at the top.